src/java.base/linux/classes/jdk/internal/platform/CgroupSubsystemFactory.java

```diff
@@ -196,7 +196,7 @@ static CgroupMetrics create() {
             if (isCgroupsV2) {
                 action = (tokens -> setCgroupV2Path(infos, tokens));
             }
-            selfCgroupLines.map(line -> line.split(":"))
+            selfCgroupLines.map(line -> line.split(":", 3))
```

Comment:
Could you please add a comment explaining that the limit value of 3 comes from the fact that `/proc/self/cgroup` contains three colon-separated tokens per line. The last value, cgroup path, might contain `:`.